src/lib/components/fund/index.tsx

```diff
@@ -74,7 +74,7 @@ export const AttachFund = ({
 }: AttachFundProps) => {
   return (
     <>
-      <Flex mb={6}>
+      <Flex mb="20px">
```

Comment:
`JsonView` only allows the line count to be changed, not the height.
<img width="1259" alt="image" src="https://user-images.githubusercontent.com/14128977/225512655-01e67998-b12c-4b08-8c0c-1b6bbf63412e.png">